sim/druid/item_sets_pve_phase_6.go

```diff
@@ -183,15 +183,33 @@ var ItemSetGenesisFury = core.NewItemSet(core.ItemSet{
 
 // Each time you Dodge while in Dire Bear Form, you gain 10% increased damage on your next Mangle or Swipe, stacking up to 5 times.
 func (druid *Druid) applyTAQGuardian2PBonus() {
-	label := "S03 - Item - TAQ - Druid - Guardian 2P Bonus"
-	if druid.HasAura(label) {
-		return
-	}
```

Comment:
This will end up breaking if using the set bonus + corresponding should rune with this change. We should leave the `HasAura` check to prevent that